.github/workflows/build.yml

```diff
@@ -4,14 +4,12 @@ on:
   push:
     branches:
       - develop
```

Comment:
There is also a master branch. Do you want to include that?

Reply:
Since it's 7 years behind, I think we should just delete it.